src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/Editor/DefaultVisualStudioCompletionBroker.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Editor.Razor;
+using Microsoft.VisualStudio.Text.Editor;
+using MonoDevelop.Ide.CodeCompletion;
+
+namespace Microsoft.VisualStudio.Mac.LanguageServices.Razor.Editor
+{
+    internal class DefaultVisualStudioCompletionBroker : VisualStudioCompletionBroker
+    {
+        public override bool IsCompletionActive(ITextView textView)
```

Comment:
@ToddGrun @KirillOsenkov would appreciate if you guys could review since you have more monodevelop knowledge. In the WebTooling code you guys don't utilize the `textView.HasAggregateFocus` so wanted your feedback.